examples/example-2/client/example-5.js

```diff
@@ -1,37 +1,33 @@
-Person = new ReactiveConstructor('Person', function() {
-  return {
-    typeStructure: [{
-      type: 'parentInstance',
-      fields: {
-        name: String,
-        child: Person,
-        friends: [ Person ]
-      }
-    }]
-  };
-});
+var testTemplate = Template.testTemplate;
+var childInstance = new ReactiveVar( false );
+
+testTemplate.onCreated(function() {
+
+  // Let's use the new Person( new Person() ) as well just to make sure it works!
 
-var grandParentInstance = new Person({
-  name: 'Super old Grandpa (so old he dead!!)',
-  child: new Person({
-    name: 'Old man Kristoffer',
-    child: new Person({
-      name: 'Papa Kristoffer',
+  var grandParentInstance = new Person(
+    new Person({
+      name: 'Super old Grandpa (so old he dead!!)',
       child: new Person({
-        name: 'Baby Boy Stoffe'
-      })
+        name: 'Old man Kristoffer',
+        child: new Person({
+          name: 'Papa Kristoffer',
+          child: new Person({
+            name: 'Baby Boy Stoffe'
+          })
+        })
+      }),
+      friends: [ new Person({ name: 'Friend 1' }), new Person({ name: 'Friend 2' })]
     })
-  }),
-  friends: [ new Person({ name: 'Friend 1' }), new Person({ name: 'Friend 2' })]
-});
+    );
 
-var childInstance = grandParentInstance.getReactiveValue('child').getReactiveValue('child').getReactiveValue('child');
+  childInstance.set( grandParentInstance.getReactiveValue('child').getReactiveValue('child').getReactiveValue('child') );
 
-var testTemplate = Template.testTemplate;
+});
 
 testTemplate.helpers({
-  childInstance: function() {
-    return childInstance;
+  childInstance() {
```

Comment:
'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).